Symfony/demo/serverless.yml

```diff
@@ -0,0 +1,44 @@
+service: symfony
+
+provider:
+    name: aws
+    region: us-east-1
+    stage: dev
+    runtime: provided.al2
+    environment:
+        APP_ENV: dev
```

Comment:
Let's switch to "prod"? (to avoid deploying something public with env variables exposed on the error page, etc.)